samples/system-test/samples.js

```diff
@@ -30,23 +30,14 @@ const PROJECT_ID = process.env.GCLOUD_PROJECT;
 const TRIGGER_ID =
   process.env.TRIGGER || 'c9033094-51a9-44c5-b3a0-1d882deb4464';
 
-const {CloudBuildClient} = require('@google-cloud/cloudbuild');
-const cb = new CloudBuildClient();
-
 describe('Sample Integration Tests', () => {
   it('should run quickstart.js', async () => {
-    execSync(
+    const stdout = execSync(
       `node ./samples/quickstart.js ${PROJECT_ID} ${TRIGGER_ID} cloud-build-mvp`,
       {cwd}
     );
-    // confirm that a build has just been kicked off.
```

Comment:
it would be nice if the generator stubbed out a `system-test`.

Reply:
We don't plan to do it for other languages, but as a feature request, let me file it: https://github.com/googleapis/gapic-generator-typescript/issues/89